src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -320,6 +319,7 @@ private static String getFullyQualifiedClassName(String filePath, String moduleN
                 "com.puppycrawl.tools.checkstyle.checks.javadoc.SummaryJavadocCheck");
         moduleMappings.put("LineLength",
                 "com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck");
+        String fullyQualifiedClassName;
```

Comment:
Love it when we find new violations within the project :)

Reply:
same here 😄